### PR TITLE
Update browser-usage-report.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the usage information stating that the report is available on any devices that uses a m365 account.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6722

File: microsoft-365/admin/activity-reports/browser-usage-report.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ description: "Learn how to get a Microsoft browser usage report using the Micros
 
 # Microsoft 365 Reports in the admin center - Microsoft browser usage
 
-The Microsoft 365 Reports dashboard shows you an activity overview across the products in your organization. It enables you to drill into individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md). In the Microsoft browser usage report, you can gain insights on Internet Explorer, Microsoft Edge Legacy, and new Microsoft Edge usage. Usage reporting is based on Microsoft 365 online services accessed by using a Microsoft browser.
+The Microsoft 365 Reports dashboard shows you an activity overview across the products in your organization. It enables you to drill into individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md). In the Microsoft browser usage report, you can gain insights on Internet Explorer, Microsoft Edge Legacy, and new Microsoft Edge usage. Usage reporting is based on Microsoft 365 online services accessed by using a Microsoft browser on any device that uses a M365 account.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/browser-usage-report.md]:
```suggestion
The Microsoft 365 Reports dashboard shows you an activity overview across the products in your organization. It enables you to drill into individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md). In the Microsoft browser usage report, you can gain insights on Internet Explorer, Microsoft Edge Legacy, and new Microsoft Edge usage. Usage reporting is based on Microsoft 365 online services accessed by using a Microsoft browser on any device that uses a Microsoft 365 account.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@sarahwxy Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@sarahwxy Gentle reminder

**Comment by yogkumgit** [comment]
@tiburd Please help us here. Thanks

**Comment by yogkumgit** [comment]
@tiburd

**Pull request merged**